src/common/motr/m0common.c

```diff
@@ -177,23 +203,33 @@ bool m0_key_iter_next(struct kvstore_iter *iter)
 	struct m0_idx *index = iter->idx.index_priv;
 	bool can_get_next = false;
 
+   perfc_trace_inii(PFT_M0_KEY_ITER_NEXT, PEM_NFS_TO_MOTR);
```

Comment:
The diff in github is showing a difference in starting whitespaces in the line. If this is genuine, pls fix them.

Reply:
fixed.